lib/pagseguro/transaction_request.rb

```diff
@@ -109,7 +109,7 @@ def create
       request = if receivers.empty?
                   Request.post('transactions', api_version, params)
                 else
-                  Request.post_xml('transactions', api_version, credentials, xml_params)
+                  Request.post_xml('transactions/', nil, credentials, xml_params)
```

Comment:
Because the API does not accept api version, so we must use:

```
https://ws.sandbox.pagseguro.uol.com.br/transactions/?
```

Not:
```
https://ws.sandbox.pagseguro.uol.com.br/v2/transactions?
```